src/as_execution/abi.rs

```diff
@@ -930,6 +930,23 @@ where
     Ok(buffer)
 }
 
+/// performs a sha256 hash on bytes and returns the hash as bytes
+#[named]
+pub(crate) fn assembly_script_sha256(mut ctx: FunctionEnvMut<ASEnv>, bytes: i32) -> ABIResult<i32> {
+    let env = ctx.data().clone();
+    sub_remaining_gas_abi(&env, &mut ctx, function_name!())?;
```

Comment:
This should not be executed if feature gas calibration is enabled

Reply:
So  `assembly_script_hash` also should be updated:

https://github.com/massalabs/massa-sc-runtime/blob/d96b37d11e0c85822131943eb310c6b2e9fafe7e/src/as_execution/abi.rs#L253-L267

Reply:
As you can see, it's already handled here, there is nothing to do

https://github.com/massalabs/massa-sc-runtime/blob/d96b37d11e0c85822131943eb310c6b2e9fafe7e/src/env/mod.rs#L87-L94

Reply:
done

Reply:
my bad. Closing it...

Reply:
done